### ISSUE TITLE
没有就ARM

### ISSUE THREAD

**Issue by MagaTailor** [Immington-Industries/rust-wlc#53]
https://github.com/rust-lang/rust/issues/29867

``` rust
编译rustwlc v0.5.0
错误[E0308]：类型不匹配
    - > .cargo /注册表/ src目录/ github.com-1ecc6299db9ec823 / rustwlc-0.5.0 / src目录/ handle.rs：644：21
    |
644 |字符= wlc_view_get_title（self.0）;
    | ^^^^^^^^^^^^^^^^^^^^^^^^^^预计I8，发现U8

错误[E0308]：类型不匹配
    - > .cargo /注册表/ src目录/ github.com-1ecc6299db9ec823 / rustwlc-0.5.0 / src目录/ handle.rs：648：39
    |
648 | pointer_to_string（字符）
    | ^^^^^预计U8，发现I8

错误[E0308]：类型不匹配
    - > .cargo /注册表/ src目录/ github.com-1ecc6299db9ec823 / rustwlc-0.5.0 / src目录/ handle.rs：657：21
    |
657 |字符= wlc_view_get_class（s​​elf.0）;
    | ^^^^^^^^^^^^^^^^^^^^^^^^^^预计I8，发现U8

错误[E0308]：类型不匹配
    - > .cargo /注册表/ src目录/ github.com-1ecc6299db9ec823 / rustwlc-0.5.0 / src目录/ handle.rs：661：35
    |
661 | pointer_to_string（字符）
    | ^^^^^预计U8，发现I8

错误[E0308]：类型不匹配
    - > .cargo /注册表/ src目录/ github.com-1ecc6299db9ec823 / rustwlc-0.5.0 / src目录/ handle.rs：670：21
    |
670 |字符= wlc_view_get_app_id（self.0）;
    | ^^^^^^^^^^^^^^^^^^^^^^^^^^^预计I8，发现U8

错误[E0308]：类型不匹配
    - > .cargo /注册表/ src目录/ github.com-1ecc6299db9ec823 / rustwlc-0.5.0 / src目录/ handle.rs：674：35
    |
674 | pointer_to_string（字符）
    | ^^^^^预计U8，发现I8

错误：放弃因6先前的错误


```

**Comment by Timidger** [comment]
Would importing `std::os::raw::c_char` fix this issue? I'm sorry, I don't have an ARM device to test this against.

**Comment by MagaTailor** [comment]
Changing the offending `*const i8`'s to `*const _` should suffice for a quick fix.

**Comment by Timidger** [comment]
Ok thanks, will make the change and it should be available with the next release (0.5.1)
